seqr/views/apis/anvil_workspace_api.py

```diff
@@ -79,6 +80,11 @@ def create_project_from_workspace(request, namespace, name):
     # Add the seqr service account to the corresponding AnVIL workspace
     add_service_account(request.user, namespace, name)
 
+    # Validate the data path
+    if not request_json['dataPath'].startswith("gs://") or not does_file_exist(request_json['dataPath']):
+        error = 'Data path is not valid.'
```

Comment:
I think it would be helpful to users if you specify in the error what the data path is. So something like `'File "{}" not found'.format(request_json['dataPath'])`